### PR TITLE
Reformat code with black

### PR DESCRIPTION
**Pull request by XVilka**
[Black](https://github.com/psf/black) is the uncompromising Python code formatter. By using it, you agree to cede control over minutiae of hand-formatting. In return, Black gives you speed, determinism, and freedom from pycodestyle nagging about formatting. You will save time and mental energy for more important matters.

Blackened code looks the same regardless of the project you're reading. Formatting becomes transparent after a while and you can focus on the content instead.

Black makes code review faster by producing the smallest diffs possible.

### CODE REVIEW

**Comment by XVilka** [comment]
Updated, also fixed some of the Codacy issues. @ivg @gitoleg please take a look.

File: src/bap/arm.py [hunk 1]
```diff
@@ -6,239 +6,873 @@
...
-
-class CPS2p(Special) : pass
-class DMB(Special) : pass
-class DSB(Special) : pass
-class HINT(Special) : pass
-class MRS(Special) : pass
-class MSR(Special) : pass
-class PLDi12(Special) : pass
-class SVC(Special) : pass
+
```
Comment (ivg) [inline src/bap/arm.py:9]: This renders the module totally unreadable. The whole idea is that you can quickly see different variants of the ADT.

File: src/bap/rpc.py [hunk 2]
```diff
@@ -117,10 +117,10 @@ def load(self):
         if self.msg is None:
             self.msg = self.bap.get_resource(self.ident)
             if not self._name in self.msg:
-                if "error" in msg:
+                if "error" in self.msg:
```
Comment (ivg) [inline src/bap/rpc.py:120]: it looks like that there was a bug here

**Review by ivg — CHANGES_REQUESTED** [review]
I am definitely against the black formatting, it destroys code readability. There is, however, a real bug fix (the second commit) which we will happily accept. 

Thanks!

**Comment by XVilka** [comment]
Closing in favor or just fixes then: https://github.com/BinaryAnalysisPlatform/bap-python/pull/15

**Pull request closed without merging**